src/phoenix/server/app.py

```diff
@@ -112,8 +115,16 @@ class Static(StaticFiles):
 
     def __init__(self, *, app_config: AppConfig, **kwargs: Any):
         self._app_config = app_config
+        self.manifest = self._load_manifest()
         super().__init__(**kwargs)
 
+    def _load_manifest(self) -> Dict[str, Any]:
+        try:
+            with open(self._app_config.manifest_json_path, "r") as f:
+                return cast(Dict[str, Any], json.load(f))
+        except (FileNotFoundError, json.JSONDecodeError) as e:
+            raise e
```

Comment:
any reason not to just raise? What's the need for the try catch?

Reply:
this was just cause i don't know python will update